static/include/components/apis/generated/base.md

```diff
@@ -164,6 +164,7 @@ For more information, see the [Flutter SDK Docs](https://flutter.viam.dev/viam_s
 ### SetPower
 
 Set the linear and angular power of the base, represented as a percentage of max power for each direction in the range of [-1.0 to 1.0].
+Supported by the micro-RDK.
```

Comment:
[optional] It might be redundant to have this at the end of every API method description if we also have the micro-RDK support column in the API tables

Reply:
That assumes that people will actually look at the table. I don't think they necessarily will. We can consider making this a badge or something like that in a future PR but let's keep as is for now